interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
Hmm interesting. Was the dodginess of our builds at all related to the `node_modules` cache? If not then it might decrease the build time if that directory is mounted as a Docker `volume` so that `npm` can use its cache.